cms-contentful/it/ContentfulCmsServiceIT.java

```diff
@@ -27,13 +27,11 @@ public void setUp() throws Exception {
         contentfulCmsService = ContentfulCmsService.of(SPACE_ID, TOKEN);
     }
 
-    @Test
+    @Test(expected=ExecutionException.class)
```

Comment:
As you are using AssertJ, there's a better way to test exceptions, check this: http://joel-costigliola.github.io/assertj/assertj-core-features-highlight.html#exception-assertion

With that, you can even check that the cause is not empty (and even better, if you know what kind of exception is inside, you can check the type).